src/app/layout.tsx

```diff
@@ -46,10 +47,13 @@ async function getAppConfig() {
 
 export default async function RootLayout({ children }: { children: ReactNode }) {
 	const appConfig = await getAppConfig();
+	const jwt = cookies().get(EGO_JWT_KEY)?.toString();
```

Comment:
https://github.com/icgc-argo/rdpc-ui/pull/110/files#diff-56ef2ffbaaa90be7e518b149c6d2a06a7560f741639aef847658fe858ebfbb2bR60 
similar functionality? One looks like it's using Next apis and the other is native?

I think the cookie return value will either be falsy or a string, so can likely remove `toString()` unless I'm missing something